doc/manual/noteworthy-differences.rst

```diff
@@ -6,6 +6,39 @@
 Noteworthy Differences from other Languages
 *******************************************
 
+Noteworthy and unusual Julia features
+---------------------------------
+
+* Julia allows identifiers with complex Unicode, for example you could represent the value of a chi-squared test with ``χ²``,
```

Comment:
It is a whole set of characters that at least as I understand the Unicode standard UAX 31, shouldn't be considered part of identifiers.  (I have no problem at all with Unicode in identifiers, I just brought up a point where it seemed inconsistent with other uses of Unicode characters in Julia)